rest-api/test/unit_test/dao_test/participant_summary_dao_test.py

```diff
@@ -292,6 +294,36 @@ def add_sample(test_code, sample_id):
     self.assertEquals(summary.numBaselineSamplesArrived, 1)
     self.assertNotEqual(init_last_modified, summary.lastModified)
 
+  def test_only_update_dna_sample(self):
+    dna_tests = ["1ED10", "1SAL2"]
```

Comment:
Add a new test case, make sure only update dna sample will not update enrollmentStatusCoreStoredSampleTime, because other criteria not meet.

Reply:
We should have a similar test that the CoreOrderedSampleTime doesn't update once it's set (except to be set to None). See above.